### DISCUSSION TITLE
Delete files from app

### DISCUSSION THREAD

**Discussion by miss14332** [btimofeev/UniPatcher#53] [Ideas]
Plss add a in-app files deleting feature . It will be very helpful for everyone . Thank you

**Comment by btimofeev** [comment]
I probably will not be able to do this, since the standard android dialog is used to select files.  And I cannot change his behavior.  I can only advise you to use an external file manager.

**Comment by miss14332** [comment]
Ohh . I guess that just how it is. anyway the app is very helpful . Hope
for more improvement. God speed.
